airgun/widgets.py

```diff
@@ -90,6 +90,57 @@ def fill(self, name):
         return self.select(name)
 
 
+class DateTime(Widget):
+    """Classical collection of paternfly data picker and two inputs for
```

Comment:
/s/data/date
And probably remove 'classical' and 'paternfly' words - i've checked patternfly, their widget looks differently, it's probably satellite's own implementation. Or am i missing something?